spring-boot/ServletReg/src/main/java/com/baeldung/servlets/springboot/embedded/EmbeddedTomcatExample.java

```diff
@@ -0,0 +1,14 @@
+package com.baeldung.servlets.springboot.embedded;
+
+import org.springframework.boot.context.embedded.EmbeddedServletContainerFactory;
+import org.springframework.boot.context.embedded.tomcat.TomcatEmbeddedServletContainerFactory;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration public class EmbeddedTomcatExample {
```

Comment:
Fixed - also fixed some broken pieces of the root app - namely a missing dependency!